app/domain/conjur/fetch_trusted_proxies.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+require 'command_class'
+
+module Conjur
+
+  FetchTrustedProxies ||= CommandClass.new(
+    dependencies: {
+        role_cls:                 ::Role,
+        validate_account_exists:  ::Authentication::Security::ValidateAccountExists.new,
+        logger:                   Rails.logger
+    },
+    inputs: %i(account)
+  ) do
+
+    def call
+      validate_account_exists
+      fetch_trusted_proxies_list
+      delete_duplications_in_list
+      trusted_proxies_list
+    rescue => e
+      raise Errors::Conjur::TrustedProxiesMissing.new(
+          host_id,
+          e.inspect
+      )
+    end
+
+    private
+
+    def validate_account_exists
+      @validate_account_exists.(
+          account: @account
+      )
+    end
+
+    def trusted_proxies_list
+      @proxies_list
```

Comment:
please, elaborate why is this function needed

Reply:
Its aligned to our code, i dont remember the motivation

Reply:
It looks like it does not do anything. Maybe you should consider to remove it

Reply:
the motivation is to let the `call` method tell a story. this way the `call` method is:
```
validate_account_exists
fetch_trusted_proxies_list
delete_duplications_in_list
trusted_proxies_list
```

it can be also:
```
validate_account_exists
fetch_trusted_proxies_list
delete_duplications_in_list
@trusted_proxies_list
```

and the method will be removed but i think there's no harm in having it